azext_edge/edge/_help.py

```diff
@@ -1020,7 +1020,8 @@ def load_iotops_help():
         "iot ops asset endpoint create custom"
     ] = """
         type: command
-        short-summary: Create an asset endpoint profile with a custom connector.
+        short-summary: Create an asset endpoint profile for a custom connector.
+        long-summary: For an example of a custom connector, please see https://aka.ms/rest-connector-quickstart
```

Comment:
Good idea but this is a private repo atm

Reply:
fml gonna remove then
- also wtf everything failed now